src/pretix/base/services/mail.py

```diff
@@ -349,6 +353,19 @@ def mail_send_task(self, *args, to: List[str], subject: str, body: str, html: st
                         logger.exception('Could not attach invoice to email')
                         pass
 
+        if attach_cached_files:
+            for cf in CachedFile.objects.filter(id__in=attach_cached_files):
+                if cf.file:
+                    try:
+                        email.attach(
+                            cf.filename,
+                            cf.file.file.read(),
+                            mimetype=None  # TODO: can we get that from cachedfile?
```

Comment:
I think there is a ``cf.type`` ;)

Reply:
Of course there is! They are important... :see_no_evil: